frontend/app/src/components/helper/display/icons/AssetIcon.vue

```diff
@@ -4,7 +4,7 @@
       <v-tooltip top open-delay="400" :disabled="noTooltip">
         <template #activator="{ on, attrs }">
           <div
-            v-if="chainIcon"
+            v-show="chainIcon"
```

Comment:
I am guessing this happens for the layout, how many undeeded elements will be drawn in this case? can't we use an empty div on v-else as a placeholder instead?

Reply:
This is common bug in vue https://github.com/vuejs/vue/issues/10608

Reply:
The question would be why would we need to change from v-if to v-show, what does this fix (is it the chain icon not appearing?)and why can't we fix it by still using v-if?

v-if would be preferable to v-show since v-show will still render something there (v-img DOM).

Reply:
The chain icon is not appear correctly.

Using `v-if`, if you have 20 data (2 pages each 10 items)
first page: first data don't have chain icon, 9 have.
then we go to second page, the chain is not show correctly (follow the first page chains)

Reply:
I'll try to look other way to fix this

Reply:
ok, so the problem we are trying to fix is the content not being reactive?
Would it work if you add a `:key` to either the div or its children?

Reply:
Yeah it actually works :), case solved